modules/factory.js

```diff
@@ -0,0 +1,51 @@
+var Factory = function(land)
```

Comment:
What bumjung means is, each factory should create a different object
ie) Food Factory, Organism Factory,